boost.sh

```diff
@@ -675,8 +675,26 @@ EOF
 generateLinuxUserConfig()
 {
     cat > "$BOOST_SRC/tools/build/src/user-config.jam" <<EOF
-using gcc : : g++ $LINUX_ARCH_FLAGS $EXTRA_LINUX_FLAGS
-: <architecture>x86 <target-os>linux
+using gcc : 6.3.0~x86_64
```

Comment:
Why are you downgrading to gcc 6.3 here?

Reply:
the initial idea was to rely on webrtc's clang, unfortunately bjam doesn't seem to cope well with it and I was unable to use clang's assembler to build `context`. GCC 6.3.0 seems to be the stable version for the debian 9 rtc-cpp uses...

Reply:
Could it harm to use our default gcc-7 or 8 instead?

Reply:
It turns out ubuntu 16.04, the one being used at the moment does not support either gcc-7-cross or gcc-8-cross. In order to move to a new cross compiler for arm, the simplest would be to update from ubuntu 16.04 to ubuntu 18.04. What do you think?